CONTRIBUTING.md

```diff
@@ -102,8 +102,17 @@ Here are some things to keep in mind as you file pull requests to fix bugs, add
 
 ### Release process
 
-- if you aren't sure if a release should happen, open an issue
-- make sure the tests pass
-- `./tools/bump.ts $NEW_VERSION`
-- `node tools/publish.js`
-- update CHANGELOG.md
+- If you aren't sure if a release should happen, open an issue
```

Comment:
question: Is this still valid? I would assume this is for maintainers and I don't think we'd use the issue tracker to discuss potential releases.

Reply:
I agree- questions would not get fielded on this platform

Reply:
Yeah, I think I would just remove this bullet; it's not particularly helpful.